extension/src/utils.js

```diff
@@ -33,8 +33,14 @@ function getLogger() {
   return logger
 }
 
+function getAuthorizationHeader(request) {
```

Comment:
again responsibility would fit better `validator/authentication-validator` also wondering the naming for  `validator/authentication-validator.js`, I would probably change it to `src/authentication.js`. 

With method names:
- `getAuthorizationHeaderValue(request)`
- _generateAuthorizationHeaderValue -> `generateBasicAuthenticationToken(commercetoolsProjectKey)`
-  isAuthorized(paymentObject, authTokenString) -> `hasValidAuthorizationHeader(commercetoolsProjectKey, authorizationHeaderValue)`

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/fee3e13ecaebc43ac39f0541d11998f9e89974d4

Reply:
Return authentication object from config instead of 3 separate attributes
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/efe72909d75312e90eaa34d61be21055e14a13dc